pandas/tests/dtypes/test_generic.py

```diff
@@ -18,9 +18,9 @@
 class TestABCClasses:
     tuples = [[1, 2, 2], ["red", "blue", "red"]]
     multi_index = pd.MultiIndex.from_arrays(tuples, names=("number", "color"))
-    datetime_index = pd.to_datetime(["2000/1/1", "2010/1/1"])
+    datetime_index = pd.to_datetime(["1/1/2000", "1/1/2010"])
     timedelta_index = pd.to_timedelta(np.arange(5), unit="s")
-    period_index = pd.period_range("2000/1/1", "2010/1/1/", freq="M")
+    period_index = pd.period_range("1/1/2000", "1/1/2010", freq="M")
```

Comment:
`pd.to_datetime(["2000/1/1", "2010/1/1"])` doesn't raise, but `pd.period_range("2000/1/1", "2010/1/1/", freq="M")` does

don't know why, need to investigate